### PR TITLE
Add placeholder entry to global.json

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/36846

JSON comments are not supported as we have consumers that don't support that.

This should eliminate the plaguing merge conflicts in global.json

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19264.13",
     "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19264.13",
+    "FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
```
Comment (joperezr) [inline global.json]: 😆

**Comment by ahsonkhan** [comment]
Lol, it's only fitting that such a PR would have merge conflicts :)
image

File: global.json (hunk 1 above)
Comment (ahsonkhan) [reply global.json]: I am curious, does 85B6 9C38 mean anything? I thought maybe you were trying to link to the issue number somehow: 36846

Comment (ViktorHofer) [reply global.json]: no, it's a random guid to avoide having a real package with that name.

**Pull request merged**